src/api/services/participantsService.ts

```diff
@@ -53,6 +58,21 @@ export const getSharingParticipants = async (participantSiteId: number): Promise
   return fetchSharingParticipants(sharingListResponse);
 };
 
+export const getParticipantsAwaitingApproval = async (email: string): Promise<Participant[]> => {
+  const approvers = await Approver.query().distinct('participantTypeId').where('email', email);
+  const approvableParticipantTypeIds = approvers.map((approver) => approver.participantTypeId);
```

Comment:
Could we move the logic of getting participantTypeIds by email to the approver service?

Reply:
done!